{"uid":"2358554bcc6d7cf7","name":"test_full_cluster_restart[SIGTERM-postgres]","fullName":"tests.integration.ha_tests.test_self_healing#test_full_cluster_restart","historyId":"1fae0f26d7819f653f6b98b8e1174bd5","time":{"start":1713405227226,"stop":1713405293823,"duration":66597},"description":"This tests checks that a cluster recovers from a full cluster restart.\n\n    The test can be called a full cluster crash when the signal sent to the OS process\n    is SIGKILL.\n    ","descriptionHtml":"<p>This tests checks that a cluster recovers from a full cluster restart.</p>\n<pre><code>The test can be called a full cluster crash when the signal sent to the OS process\nis SIGKILL.\n</code></pre>\n","status":"failed","statusMessage":"AssertionError: Not all units down at the same time.\nassert False","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 849, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 883, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 256, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 182, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 172, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1772, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 195, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/tests/integration/ha_tests/test_self_healing.py\", line 274, in test_full_cluster_restart\n    assert await are_all_db_processes_down(\nAssertionError: Not all units down at the same time.\nassert False","flaky":true,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop","time":{"start":1713403567463,"stop":1713403567464,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1713403567464,"stop":1713403567607,"duration":143},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1713403567464,"stop":1713403567464,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1713405218222,"stop":1713405218222,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1713405218222,"stop":1713405218222,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"reset_restart_condition","time":{"start":1713405218222,"stop":1713405226184,"duration":7962},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"loop_wait","time":{"start":1713405226184,"stop":1713405227225,"duration":1041},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"loop_wait::finalizer","time":{"start":1713405293825,"stop":1713405293879,"duration":54},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"reset_restart_condition::finalizer","time":{"start":1713405293879,"stop":1713405601712,"duration":307833},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes::finalizer","time":{"start":1713405601713,"stop":1713405607749,"duration":6036},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1713405607750,"stop":1713405607750,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1713408361655,"stop":1713408361931,"duration":276},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1713408361932,"stop":1713408361932,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1713408361932,"stop":1713408361932,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1713408361933,"stop":1713408361933,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"fv-az661-801"},{"name":"thread","value":"7084-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"process","value":"'postgres'"},{"name":"signal","value":"'SIGTERM'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":0,"skipped":0,"passed":4,"unknown":0,"total":6},"items":[{"uid":"b48cc6f03f4de98","reportUrl":"../13//#testresult/b48cc6f03f4de98","status":"passed","time":{"start":1713319241360,"stop":1713319278094,"duration":36734}},{"uid":"1e4a8b1a614c3613","reportUrl":"../8//#testresult/1e4a8b1a614c3613","status":"passed","time":{"start":1713232551007,"stop":1713232582146,"duration":31139}},{"uid":"783b8dce5a3cfb50","reportUrl":"../7//#testresult/783b8dce5a3cfb50","status":"passed","time":{"start":1713151482148,"stop":1713151523035,"duration":40887}},{"uid":"d6038edbf1cdd13e","reportUrl":"../4//#testresult/d6038edbf1cdd13e","status":"passed","time":{"start":1712973349790,"stop":1712973376128,"duration":26338}},{"uid":"519602d2389d4a9d","reportUrl":"../2//#testresult/519602d2389d4a9d","status":"failed","statusDetails":"AssertionError: Not all units down at the same time.\nassert False","time":{"start":1712886796994,"stop":1712886862215,"duration":65221}}]},"tags":["asyncio","group(1)"]},"source":"2358554bcc6d7cf7.json","parameterValues":["'postgres'","'SIGTERM'"]}